test/distributed/tensor/parallel/test_tp_examples.py

```diff
@@ -209,11 +209,14 @@ def test_transformer_training(self, is_seq_parallel=False):
                 "pos_embeddings": ColwiseParallel(output_layouts=Replicate()),
             }
 
-        model_tp = parallelize_module(
-            model_tp,
-            device_mesh,
-            root_plan
-        )
+        with CommDebugMode() as comm_mode:
+            model_tp = parallelize_module(
```

Comment:
Ohh I think we want to track the collectives in runtime forward/backward, we don't need to track the sharding part (parallelize_module)